packages/babel-template/src/formatters.js

```diff
@@ -54,16 +54,16 @@ export const statement: Formatter<BabelNodeStatement> = makeStatementFormatter(
 
 export const expression: Formatter<BabelNodeExpression> = {
   code: str => `(\n${str}\n)`,
-  validate: (ast: BabelNodeFile) => {
-    const { program } = ast;
+  validate: ({ program }) => {
     if (program.body.length > 1) {
       throw new Error("Found multiple statements but wanted one");
     }
-    const expression = program.body[0].expression;
-    if (expression.start === 0) {
+
+    if (program.body[0].expression && program.body[0].expression.start === 0) {
```

Comment:
want previous version more readable? would ofc need to add `expression &&` to the if's condition

Reply:
For sure it was, this is just not great workaround for flow :(

Similarly to unwrap below it, `const expression = program.body[0].expression;` trips up a bunch of `Property not found` errors due to `program.body[0]` being `Array<BabelNodeStatement>`.

Reply:
Not sure if I understand the problem here, I guess I'd have to check out produced errors but it seems that those 2 should be equivalent and that they shouldnt trip flow:
```js
if (program.body[0].expression && program.body[0].expression.start === 0)
```
vs
```js
const expression = program.body[0].expression;
if (expression && expression.start === 0)
```

Reply:
@Andarist I don't disagree :)

Reply:
Might be better to just revert this change and just mark this as a FIXME as well (like unwrap)?

Reply:
i'd say it's better to keep it "fixed" (like it is now) - Im only confused why those 2 are not equivalent

Reply:
Yeah, I'll try to do more digging later today.

Reply:
There's a lot of other stuff in this PR that would be good to land. I'm fine with `$FlowFixMe` if it'll let us land this.